miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultContainerService.java

```diff
@@ -120,7 +119,6 @@ public SequencerPartitionContainer save(SequencerPartitionContainer container) t
   @Override
   public void applyChanges(SequencerPartitionContainer source, SequencerPartitionContainer managed) throws IOException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 21 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)